### ISSUE TITLE
Camera feed is on whenever the extension opens

### ISSUE THREAD

**Issue by jak-pan** [polkadot-js/extension#1492]
The extension is starting camera feed whenever it opens. This should only happen when you are asking to confirm QR code in one single step. Not when browsing accounts or signing non-qr transactions.

**Comment by jak-pan** [comment]
> I can't say this has ever happened to me, what is your enviornment/browser?

MacOs 15, 14 and any of the latest ~6+ months of Chrome. It has done this forever, since I remember, confirmed by colleagues and other ecosystem agents. I just got triggered by the spying notif that I reported it yesterday, since it was not in the issue list. 😃 

If you click this allow QR code camera feed, the camera feed instantly starts and runs all the time extension is open.
<img width="327" alt="Screenshot 2024-12-24 at 12 06 06" src="https://github.com/user-attachments/assets/53ede48c-c08b-4326-a5aa-8af2c8fc7dec" />

**Comment by Tbaut** [comment]
yeah it's happened to me too since forever, I'm on Linux Mint, Firefox. I've see the LED of my webcam go on too, when I didn't expect it. I just tested though, it's not happening right now say when I'm on pjs apps, and either opening the extension popup, or signing a payload with an account that's not a Polkadot Vault account.
I wonder if there's a way to consistently reproduce, I'll keep an eye for it
